src/libraries/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -309,6 +262,84 @@
     <Compile Include="$(CommonPath)System\Text\ValueStringBuilder.cs"
              Link="Common\System\Text\ValueStringBuilder.cs" />
   </ItemGroup>
+  <!-- Header support -->
```

Comment:
All of this is included for the browser build?

Reply:
You are referring to the Header support right?

If it is the header support then I actually do not know to be honest but `HttpMethod` and KnownHeader(s)` files reference the HPack and the QPack modules which reference the H3 and H2 static tables.  Those I have not dug into to unravel honestly.  I included that to at least be able to build something.

Reply:
I see. Ok for now then, but it sounds like we have some cleanup to do.

Reply:
I do not mind doing the clean up on my end but prefer to do that in another PR if that is possible.

Reply:
Yeah, that's fine.  I wasn't suggesting it be done in this PR or even that you do it :)